client/package.json

```diff
@@ -30,13 +30,13 @@
     "vuex-class": "^0.3.2"
   },
   "devDependencies": {
-    "@types/jest": "^24.9.0",
+    "@types/jest": "^25.1.4",
```

Comment:
Any impact going from v24 to v25?

Reply:
Besides avoiding some type errors in Jest testing, not that i noticed!